Makefile

```diff
@@ -183,6 +183,19 @@ test-in-docker:
 run-functional-tests: testnnp test-registry ecr-execution-role-image telemetry-test-image
 	. ./scripts/shared_env && go test -tags functional -timeout=30m -v ./agent/functional_tests/...
 
+.PHONY: build-image-for-ecr ecr-execution-role-image-for-upload upload-images replicate-images
+
+build-image-for-ecr: netkitten volumes-test squid awscli image-cleanup-test-images fluentd taskmetadata-validator testnnp container-health-check-image telemetry-test-image ecr-execution-role-image-for-upload
+
+ecr-execution-role-image-for-upload:
+	$(MAKE) -C misc/ecr-execution-role-upload $(MFLAGS)
+
+upload-images: build-image-for-ecr
+	@./scripts/upload-images $(STANDARD_REGION) $(STANDARD_REPOSITORY)
+
+replicate-images: build-image-for-ecr
```

Comment:
what do you think about pulling images from $STANDARD_REGION and pushing them to $REPLICATE_REGION?

That way, we have identical images.